cmd/telegraf/telegraf_windows.go

```diff
@@ -84,12 +84,17 @@ func runAsWindowsService(inputFilters, outputFilters, aggregatorFilters, process
 	// Handle the --service flag here to prevent any issues with tooling that
 	// may not have an interactive session, e.g. installing from Ansible.
 	if *fService != "" {
-		if *fConfig != "" {
-			svcConfig.Arguments = []string{"--config", *fConfig}
+		if len(fConfigs) > 0 {
+			svcConfig.Arguments = []string{}
```

Comment:
this looks like it's not necessary. append doesn't need initialization and the 0 len case does not hit this.

Reply:
I think this is not initialization, its clearing the default config from here: https://github.com/influxdata/telegraf/blob/f26bfe6755a1c5c8750bdece387ddece56838e9c/cmd/telegraf/telegraf_windows.go#L71

If a user specifies config files via command line options we should use those config files only, not the default + their configs.